package.json

```diff
@@ -9,7 +9,16 @@
     "commander": "^2.20.0",
     "lerna": "^3.20.2",
     "semver": "^7.3.2",
-    "shelljs": "^0.8.5"
+    "shelljs": "^0.8.5",
+    "eslint": "7.31.0",
+    "eslint-config-prettier": "4.0.0",
+    "eslint-import-resolver-webpack": "0.10.0",
+    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-import": "2.23.4",
+    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-prettier": "3.0.1",
+    "eslint-plugin-react": "7.24.0",
+    "prettier": "^1.18.2"
```

Comment:
@alexvuong we both agree that the solution is to somehow have these packages be available in the root (in particular, the node_modules folder at the root). However, I think the way we have it now would make it easy to forget to update these version numbers in the future. Now we need to maintain 2 sets of eslint-related dependencies: at the root and inside pwa-kit-build package.

My suggestion is: there's actually another way to get the same result, and it would involve `lerna` and its [hoisting](https://github.com/lerna/lerna/tree/main/commands/bootstrap#--hoist-glob) capability.

```js
// Inside 'scripts/bootstrap/js', add the `--hoist` flag
const cmd = `npm run lerna -- bootstrap ${commandArgs} ${environmentArgs} --hoist '+(eslint*|prettier)'`
```

What the above would do is to hoist the eslint-related packages to the root folder. Give that a try:
1. Run `npm ci` at the root
2. Automatically our bootstrap.js script gets executed
3. Look for [root]/node_modules/eslint* packages